cli/src/torii/mod.rs

```diff
@@ -50,14 +50,14 @@ pub enum Error {
     /// Failed to accept transaction
     AcceptTransaction(#[from] iroha_core::tx::AcceptTransactionFail),
     /// Error while getting or setting configuration
-    Config(#[source] eyre::Report),
+    Config(eyre::Report),
```

Comment:
Can you remind the exact problem with that? Because I think now the error message will loose it's source error

Reply:
@DCNick3 can you comment on this?

Reply:
Yeah, I think I mistakenly did something like this, confusing `#[from]` with `#[source]`

`#[from]` implies `#[source]`, but not the other way around. A `#[source]` on `eyre::Report` is fine, `#[from]` is not

Reply:
@DCNick3 , so we can keep `#[source]` here, right?

Reply:
Yes, it should be kept.